core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -143,6 +143,16 @@ public String toString() {
           .setDefaultValue(String.format("${%s}/logs", Name.WORK_DIR))
           .setDescription("The path to store log files.")
           .build();
+  public static final PropertyKey LOGS_SERVER_ENABLED =
```

Comment:
it might be good to have a more informative name to tell the difference between new `LOGS_SERVER` v.s. traditional local `LOGS` ?